test/helpers/constants.js

```diff
@@ -7,6 +7,7 @@ const EPOCH_LENGTH = BigNumber.from(40);
 const NUM_BLOCKS = 10;
 const NUM_STATES = BigNumber.from(4);
 const STATE_LENGTH = BigNumber.from(10);
+const BLOCK_REWARD = BigNumber.from(40).mul((BigNumber.from(10).pow(BigNumber.from(18))));
```

Comment:
> BigNumber.from(40).mul((BigNumber.from(10).pow(BigNumber.from(18))));

Can be re-written as `BigNumber.from(40).mul(ONE_ETHER);`

Reply:
Done, Thanks